tools/cvm-image-rewriter/tdx-libvirt-ubuntu-host.xml.template

```diff
@@ -1,10 +1,10 @@
 <domain type='kvm' xmlns:qemu='http://libvirt.org/schemas/domain/qemu/1.0'>
-  <name>td-guest-grub-on-ubuntu-host</name>
-  <memory unit='KiB'>2097152</memory>
+  <name>tdx-guest</name>
+  <memory unit='GiB'>mem_size</memory>
```

Comment:
How about changing the pattern to CAPITAL to prevent the regex matches the libvirt keywords?

Reply:
Updated with capital keywords "REPLACE_MEM_SIZE" and "REPLACE_VCPU_NUM". Thanks.